utility/utils.py

```diff
@@ -1479,3 +1479,14 @@ def fetch_image_tag(rhbuild):
         raise TestSetupFailure("Not a live testing")
     except Exception as e:
         raise TestSetupFailure(f"Could not fetch image tag : {e}")
+
+
+def validate_conf(conf):
+    """
+    Collects the conf and checks for unique ID for nodes
+    """
+    for cluster in conf.get("globals"):
+        nodes = cluster.get("ceph-cluster").get("nodes")
+        nodes_id = [node.get("id") or f"node{idx+1}" for idx, node in enumerate(nodes)]
```

Comment:
I think we need a logic to handle list and dict items considering baremetal and IBM/PSI node config.